apps/condo/domains/user/schema/User.js

```diff
@@ -205,7 +207,15 @@ const User = new GQLListSchema('User', {
             }
         },
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
+    plugins: [
+        uuided(),
+        versioned(),
+        tracked(),
+        softDeleted(),
+        dvAndSender(),
+        historical(),
+        webHooked(webhookModelValidator),
```

Comment:
Is `OK` to call it like that? Or may be add wrapper to combine validator with plugin?

Reply:
OUTDATED